smoke_zephyr/configuration.py

```diff
@@ -49,8 +49,8 @@
 
 SERIALIZER_DRIVERS = {}
 """The serializer drivers that are available."""
-SERIALIZER_DRIVERS['json'] = {'load': json.load, 'dumps': lambda obj: json.dumps(obj, sort_keys=True, indent=4)}
-SERIALIZER_DRIVERS['jsn'] = {'load': json.load, 'dumps': lambda obj: json.dumps(obj, sort_keys=True, indent=4)}
+SERIALIZER_DRIVERS['json'] = {'load': json.load, 'loads': json.loads, 'dumps': lambda obj: json.dumps(obj, sort_keys=True, indent=4)}
+SERIALIZER_DRIVERS['jsn'] = {'load': json.load, 'loads': json.loads, 'dumps': lambda obj: json.dumps(obj, sort_keys=True, indent=4)}
 if has_yaml:
 	SERIALIZER_DRIVERS['yaml'] = {'load': lambda file_obj: yaml.load(file_obj, Loader=Loader), 'dumps': lambda obj: yaml.dumps(obj, default_flow_style=False, Dumper=Dumper)}
```

Comment:
Because the JSON driver got a "loads" attribute defined, the YAML driver needs one too.